source/clear-linux/get-started/bare-metal-install/cgdisk-manual-install.rst

```diff
@@ -0,0 +1,303 @@
+
+   .. code-block:: console
+
+      First sector:  press :kbd:`Enter` to select the default value
+      Size in sectors:  4G
+      Hex code or GUID:  8200
+      Enter new partition name:  swap
+
+   Your :command:`cgdisk` partition list should now look like figure 9.
+
+   .. figure:: figures/cgdisk-manual-install-9.png
+      :scale: 50 %
+      :alt: cgdisk - swap partition defined
+
+      Figure 9: :guilabel:`cgdisk - swap partition defined`
+
+Create Linux filesystem partition
+*********************************
+
+The final partition that you will create is the Linux filesystem partition to
+be used as the root mount point for you |CL| installation.
+
+#. Highlight the largest free space entry at the bottom of the list and select
+   the :guilabel:`[ New ]` button or press the :kbd:`N` key and enter the
+   following values to create the Linux filesystem partition:
+
+   .. code-block:: console
+
+      First sector:  press :kbd:`Enter` to select the default value
+      Size in sectors:  press :kbd:`Enter` to select the default value, which
+                        will be the remainder of available space on the disk
+      Hex code or GUID:  8300
+      Enter new partition name:  root
+
+   With all the partitions now defined, you should see a list similar to what
+   is shown in figure 10:
+
+   .. figure:: figures/cgdisk-manual-install-10.png
+      :scale: 50 %
+      :alt: cgdisk - defined partitions
+
+      Figure 10: :guilabel:`cgdisk - defined partitions`
+
+#. If you are satisfied that the partition scheme is correct, you will need to
+   write this GPT to the hard drive. Select the :guilabel:`[ Write ]` button
+   or press the :kbd:`W` key and you will be prompted with:
+
+   .. code-block:: console
+
+      Are you sure you want to write the partition table to disk? (yes or no)
+
+#. Enter ``yes`` and press :kbd:`Enter` to write this data to the hard drive
+   and then select the :guilabel:`[ Quit ]` button or press :kbd:`Q` to exit
+   the :command:`cgdisk` utility and return to the |CL| manual installation
+   process.
+
+   You will see the partitions that you just created as shown in figure 11 and
+   ready for the next step in the |CL| installer setup process.
+
+#. Move your cursor to the :guilabel:`< Next >` button and press :kbd:`Enter`.
+
+   .. figure:: figures/cgdisk-manual-install-11.png
+      :scale: 50 %
+      :alt: defined partitions
+
+      Figure 11: :guilabel:`defined partitions`
+
+Set mount points
+****************
+
+The :guilabel:`Set mount points` menu will set the mount points that the |CL|
+installer will use for your |CL| installation and is shown in figure 12.
+
+.. figure:: figures/cgdisk-manual-install-12.png
+   :scale: 50 %
+   :alt: Set mount points
+
+   Figure 12: :guilabel:`Set mount points`
+
+In this menu you will need to set the mount points for the boot and root
+partitions and select to format them.
+
+#. Highlight the EFI System partition type menu entry and press the

```

Comment:
Split this step into multiple steps. 1. Select EFI partition, 2. enter the information, 3. Enable formatting partition with figure.

Reply:
done